Gemfile

```diff
@@ -5,7 +5,7 @@ gemspec
 
 group :development, :test do
   gem 'coveralls', require: false
-  gem 'i18n-tasks'
+  gem 'i18n-tasks', '0.9.21'
```

Comment:
Why such pessimism in versions?

Reply:
The new version of `i18n-tasks` breaks our current translation process. This is the version we are using now. Locking to it buys us time to update the process.

Note this only impacts development users trying to run translations through the google translate API.

In other words, is it really pessimism if the next version is actually broken? :)

Reply:
So, `.22`, `.23`, & `.24` are all broken?  Can you add a comment to that effect here?